src/geocoding.js

```diff
@@ -0,0 +1,134 @@
+import * as Cookie from './cookie';
+import * as Http from './http';
+import Navigator from './navigator';
+
+// consts
+import STATUS from './status_codes';
+
+const DEFAULT_HOST = 'https://api.radar.io';
+
+class Geocoding {
```

Comment:
I like this pattern. Lets create a `/api` directory for each of the classes to go into.

Reply:
sure thing, was thinking about something like `/services` - `/api` works well too